Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -257,20 +238,17 @@ def url_reputation_command(client: Client, args: Dict[str, Any], api_key) -> Com
             dbot_score=dbot_score
         )
 
-        url_standard_list.append(url_standard_context)
-
-        url_context_excluded_fields = ['objects', 'nir']
-        url_data_list.append({k: url_data[k] for k in url_data if k not in url_context_excluded_fields})
-
-    readable_output = tableToMarkdown('URL List', url_data_list)
-
-    return CommandResults(
-        readable_output=readable_output,
-        outputs_prefix='Pulsedive.URL',
-        outputs_key_field='indicator',
-        outputs=url_data_list,
-        indicators=url_standard_list,
-    )
+        url_data.pop('objects')
+        url_data.pop('nir')
+        command_results.append(CommandResults(
+            readable_output=tableToMarkdown('URL List', url_data),
```

Comment:
Given a single URL, I feel that URL List is not a good human-readable title, perhaps URL Details?